### DISCUSSION TITLE
Welcome to PropagandaTracker Discussions!

### DISCUSSION THREAD

**Discussion by hghalebi** [hghalebi/PropagandaTracker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChadiHelwe** [comment]
Hello,

My name is Chadi; I am a Ph.D. candidate at Télécom Paris - Institut Polytechnique de Paris. In my thesis, I am investigating methods to combine symbolic and neural approaches for textual logical reasoning.

I like the initiative to develop tools to detect propaganda bots.

**Reply by hghalebi** [reply]
You are more than welcome 👍🏽
